README.md

```diff
@@ -1,44 +1,92 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+# Covid-19-tracker
 
-## Available Scripts
 
-In the project directory, you can run:
 
-### `npm start`
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Description
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
+> "covid-19-tracker" is a webapp that allows users to track and watch the statistics of covid-19 cases for each country and also the regions within the country updated evrey day.
 
-### `npm test`
+## Screenshots
```

Comment:
- [ ] I recommend removing the `screenshots` title if the screenshots are not available so your Readme is documented professionally.